File: back-end/src/main/java/com/huyphan/dtos/NotificationDto.java
package com.huyphan.dtos;

import com.huyphan.models.enums.NotificationType;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class NotificationDto {

    private Long id;

    private NotificationType type;

    private String destUrl;

    private Boolean isViewed;

    private String created;
}

File: client/src/services/dtos/notification-dto.ts
import { NotificationType } from '../../models/enums/notification-type';
import { UserDto } from './user-dto';

export default interface NotificationDto {
  readonly id: number;

  readonly type: NotificationType;

  readonly destUrl: string;

  readonly sender: UserDto;

  readonly isViewed: boolean;

  readonly created: string;

  readonly message: string;
}
